src/data/vault.json

```diff
@@ -41,9 +41,9 @@
 	},
 	"alertBannerActive": true,
 	"alertBanner": {
-		"tag": "HashiConf Global",
-		"url": "https://hashiconf.com/global/?_gl=1*u898mc*_ga*Nzg2MzExNjMwLjE2NDQ5NDYzMDU.*_ga_P7S46ZYEKW*MTY1OTk4NjE2NC4xMjIuMS4xNjU5OTg5MjIxLjA.",
-		"text": "Join us for HashiConf Global October 4-6 in Los Angeles & online.",
+		"tag": "Just announced",
+		"url": "https://hashiconf.com/globalhttps://hashiconf.com/global",
```

Comment:
```suggestion
		"url": "https://hashiconf.com/global",
```

Reply:
Whoa, not sure how that got there 😅 Good catch!